src/main/java/com/minecolonies/network/messages/RecallCitizenMessage.java

```diff
@@ -24,10 +23,10 @@
  *
  * @author Colton
  */
-public class RecallCitizenMessage implements IMessage, IMessageHandler<RecallCitizenMessage, IMessage>
+public class RecallCitizenMessage extends AbstractMessage<RecallCitizenMessage, IMessage>
 {
     private static final double MIDDLE_BLOCK_OFFSET = 0.5D;
-    private int colonyId;
+    private int      colonyId;
     private BlockPos buildingId;
 
     public RecallCitizenMessage() {}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) It is a good practice to call super() in a constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ACallSuperInConstructor)